go.mod

```diff
@@ -13,15 +13,36 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/fsnotify/fsnotify v1.4.7 // indirect
```

Comment:
hmm wonder if you need to run a `go mod tidy` to clean these additions up

Reply:
Not sure I understand. I will look into what go mod tidy does

Reply:
Ok I see what happened here. Is running go mod tidy something you do often?

Reply:
definitely if there is stuff added into `go.mod` but there is no new imports in any of the files, it's a pretty good signal that `go mod tidy` should be run.